ablog/commands.py

```diff
@@ -470,7 +476,8 @@ def ablog_deploy(
             run("git pull", echo=True)
         else:
             run(
-                "git clone https://github.com/{0}/{0}.github.io.git {1}".format(
+                "git clone " + ("https://github.com/" if github_is_http else "git@github.com:") +
+                "{0}/{0}.github.io.git {1}".format(
```

Comment:
```suggestion
                "{0}/{0}.github.io.git {1}".format(
```

It has been a long time since I cloned a github page site. Does github.io.git resovle to a repo?

Reply:
Yes...it's the site that's specified by their instructions for a GitHub page

Reply:
That's fine!